doc/requirements/management

```diff
@@ -0,0 +1,134 @@
+---
+group: Management
+---
+
+Management operations allows the user to control and maintain Open CAS.
+They are performed from console line with `casadm` management tool.
```

Comment:
`casctl` can be used as well. I would avoid particular names because if the tools would be renamed/substituted or the management toolset would be extended with a additional tool, the document would require updating.